app/models/index-card.ts

```diff
@@ -44,6 +68,39 @@ export default class IndexCardModel extends Model {
         }
         return '';
     }
+
+    @dropTask
+    @waitFor
+    async getOsfModel() {
+        const identifier = this.resourceIdentifier;
+        if (identifier && this.osfModelType) {
+            const guid = this.guidFromIdentifierList(identifier);
+            const relatedCounts = this.osfModelType === 'user' ? 'nodes,registrations,preprints' : '';
+            if (guid) {
+                const osfModel = await this.store.findRecord(this.osfModelType, guid, {
+                    adapterOptions: {
+                        query: {
+                            related_counts: relatedCounts,
+                        },
+                    },
+                    reload: true,
```

Comment:
could this `options` hash be passed in by the caller? seems like `related_counts` and `reload` are both specific to this first usage, not something you'd expect to always want when getting a model instance

Reply:
Ahh yes! Will do that now